operator/standalone/controller.go

```diff
@@ -54,12 +62,9 @@ func (r *PostgresStandaloneReconciler) Reconcile(ctx context.Context, request re
 	if !obj.DeletionTimestamp.IsZero() {
 		return r.Delete(ctx, obj)
 	}
-	if !controllerutil.ContainsFinalizer(obj, finalizer) {
-		controllerutil.AddFinalizer(obj, finalizer)
+	if readyCondition := meta.FindStatusCondition(obj.Status.Conditions, conditions.TypeReady); readyCondition == nil {
+		// Ready condition is not present, it's a fresh object
 		res, err := r.Create(ctx, obj)
```

Comment:
Why do we have a Create method here? Isn't the object already created?

Reply:
Well, the object is created yes, but the instance is not really created (aka the postgresql service is not deployed yet). This `Create` doesn't refer to the CRD creation, but to the first-time deployment of the DB.
I'm mostly taking the same terminology as Crossplane (https://github.com/crossplane/crossplane-runtime/blob/master/pkg/reconciler/managed/reconciler.go#L274) but I'm open for other names.

Reply:
it would be helpful if the method would not create confusion. Can we change the name of the method or class to include `postresql`?

Reply:
renamed